CI/Style-To-Repo/deploy.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# This file is part of ILIAS, a powerful learning management system
+# published by ILIAS open source e-Learning e.V.
+#
+# ILIAS is licensed with the GPL-3.0,
+# see https://www.gnu.org/licenses/gpl-3.0.en.html
+# You should have received a copy of said license along with the
+# source code, too.
+#
+# If this is not the case or you just want to try ILIAS, you'll find
+# us at:
+# https://www.ilias.de
+# https://github.com/ILIAS-eLearning
+#
+# This script compares the actual style repo with the built style folder and pushes the possible changes to repo.
+
+NOW=$(date +'%d.%m.%Y %I:%M:%S')
+DEPLOY_BASE_FOLDER="./CI/Style-To-Repo/repo"
+STYLE_REPO="<complete url>"
+STYLE_REPO_NAME_SHORT="<all after github.com>"
+STYLE_REPO_USER_NAME="<user_name>"
+
+function deploy() {
+  MSG=${1}
+  HASH=${2}
+  URL=${3}
+  BRANCH=${4}
+  REPO_TOKEN="https://${5}@github.com/${STYLE_REPO_NAME_SHORT}"
+
+  if [ -d ${DEPLOY_BASE_FOLDER} ]
+  then
+    rm -rf ${DEPLOY_BASE_FOLDER}
+  fi
+
+  mkdir -p ${DEPLOY_BASE_FOLDER}
+  git clone ${STYLE_REPO} ${DEPLOY_BASE_FOLDER} >/dev/null 2>&1
```

Comment:
`git clone` needs to use **${REPO_TOKEN}** as well (instead of **${STYLE_REPO}**), as we don't have SSH available for cloning.